src/client/tests/SessionTest.cpp

```diff
@@ -243,6 +243,21 @@ TEST_F(SessionTest, JsonResult) {
   b.wait();
 }
 
+TEST_F(SessionTest, DurationResult) {
+  nebula::ConnectionPool pool;
+  nebula::Config c{10, 0, 10, 0, "", false};
+  pool.init({kServerHost ":9669"}, c);
+  auto session = pool.getSession("root", "nebula");
+  ASSERT_TRUE(session.valid());
+
+  auto resp = session.execute("YIELD duration({years: 1, seconds: 2}) AS d");
+  ASSERT_EQ(resp.errorCode, nebula::ErrorCode::SUCCEEDED);
+
+  nebula::DataSet expected({"d"});
```

Comment:
Could you add some toString test of duration?

Reply:
All code in common already tested in nebula, if want it. We could copy it to client. WDYT?

Reply:
It's ok to me. But I still don't find toString test of duration in common of nebula.